playbooks/install_stack.yaml

```diff
@@ -35,6 +35,32 @@
         ha_env:
         - /usr/share/openstack-tripleo-heat-templates/environments/docker-ha.yaml
 
+  - name: Enable OVN DVR HA by default when deploying DCN
+    when: dcn_az is defined
+    block:
+    - name: Add OVN DVR HA services
+      set_fact:
+        service_envs: "{{ service_envs | union(ovn_dvr_ha_env) }}"
+      vars:
+        ovn_dvr_ha_env:
+        - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovn-dvr-ha.yaml
+
+  - name: Enable exported environments from central site when deploying DCN
+    when:
+      - dcn_az is defined
+      - dcn_az != "central"
+    block:
+    - name: Add DCN environments
+      set_fact:
+        service_envs: "{{ service_envs | union(dcn_env) }}"
+      vars:
+        dcn_env:
+        - /opt/exported-data/passwords.yaml
```

Comment:
What copies the `exported-data` directory to `/opt`? From what I understand `export-dcn.sh` generates `/home/stack/exported-data` on the central site and we're missing a step to copy the directory to the remote sites.

Reply:
it's done by ci-configs/provision.sh in https://github.com/shiftstack/ci-configs/pull/17 - not ideal I think it's ok for now and our needs. I'll add some doc in the README about this manual step.